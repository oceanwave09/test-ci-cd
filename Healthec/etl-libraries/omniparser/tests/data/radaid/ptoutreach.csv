batch_type,First Name,Last Name,MRN/ID,Outreach type,PN_contact,PN_Contact_date,PN_barrier,Outcome
2,First26351,Last26351,26351,Breast,1,10-15-2023,1,1
2,First26352,Last26352,26352,Cervical,0,09-18-2014,9,6
2,First26353,Last26353,26353,Unknown,1,12-15-2001,5,4