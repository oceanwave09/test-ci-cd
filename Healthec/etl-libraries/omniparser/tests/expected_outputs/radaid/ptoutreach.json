[
	{
		"batch_type": "2",
		"first_name": "9e7a7d800f451bef2a6bb9e2024d7569",
		"last_name": "74967d45b9d2cfa01bd08352d5ae7802",
		"mrn": "1c26ac8bfc15d08afbccac2ab1ec40fd",
		"outcome": "1",
		"outreach_type": "Breast",
		"pn_barrier": "1",
		"pn_contact": "1",
		"pn_contact_date": "10-15-2023",
		"row_id": "2077321a-1e56-475f-b0e3-5fb1715df6a8"
	},
	{
		"batch_type": "2",
		"first_name": "8f5a96dc34bd18e95f0fae1ef8328649",
		"last_name": "137fa153590188b32101b22447adbff0",
		"mrn": "3d7c0ad3e51ca2d9fa55d2c374bcd17a",
		"outcome": "6",
		"outreach_type": "Cervical",
		"pn_barrier": "9",
		"pn_contact": "0",
		"pn_contact_date": "09-18-2014",
		"row_id": "528dc18c-1832-4690-8f8a-a4d5db124736"
	},
	{
		"batch_type": "2",
		"first_name": "7e0453536700a8d9f19f49441f1a1bab",
		"last_name": "68f3b3f9d7a7f21efee846b1a19cda3b",
		"mrn": "4d94e698dee0e06c55c995dc6f59ea55",
		"outcome": "4",
		"outreach_type": "Unknown",
		"pn_barrier": "5",
		"pn_contact": "1",
		"pn_contact_date": "12-15-2001",
		"row_id": "73d37664-bb31-496e-90d2-5e380b0ef17e"
	}
]
